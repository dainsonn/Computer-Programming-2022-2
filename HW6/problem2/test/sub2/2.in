root
pivot@eee
post
my name is
yes
\n
root and
Nice to meet you!

exit
