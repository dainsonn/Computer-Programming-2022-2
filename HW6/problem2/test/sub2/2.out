------ Authentication ------
id : passwd : ----------------------------
root@sns.com
post : Post contents
recommend <number> : Recommend <number> interesting posts
search <keyword1> ... <keywordN> <threshold> : List post entries whose contents contain keywords more than the threshold
exit : Terminate this program
----------------------------
Command : ----------------------------
New Post
* Title : * Advertising (yes/no) : * Content
> > > > ----------------------------
root@sns.com
post : Post contents
recommend <number> : Recommend <number> interesting posts
search <keyword1> ... <keywordN> <threshold> : List post entries whose contents contain keywords more than the threshold
exit : Terminate this program
----------------------------
Command : 